source/app/service-providers/commands/export.ts

```diff
@@ -74,7 +75,23 @@ export default class Export extends ZettlrCommand {
       if (fileDescriptor !== null) {
         exporterOptions.sourceFiles.push(fileDescriptor)
         exporterOptions.cwd = fileDescriptor.dir
-        exporterOptions.targetDirectory = (exportTo === 'temp') ? app.getPath('temp') : fileDescriptor.dir
+        switch (exportTo) {
+          case 'ask': {
+            const focusedWindow = BrowserWindow.getFocusedWindow()
+            const folderSelection = await this._app.windows.askDir(trans('system.export_dialog.title'), focusedWindow, trans('system.export_dialog.save'))
+            if (folderSelection === undefined || folderSelection.length === 0) {
+              getServiceContainer()?.log.error('[Export] Could not run exporter: Folderselection did not have a result!')
+              return
+            }
+            exporterOptions.targetDirectory = folderSelection[0]
```

Comment:
Does `askDir` return a list of strings …? I thought I already captured that? If not, disregard this comment!